modules/arm_plugin/src/arm_converter/arm_converter_interpolate.cpp

```diff
@@ -78,24 +75,37 @@ void wrap_interpolate(const T* input_data,
                                                 attrs);
 }
 
+void get_axes_vec(const opset::Interpolate& node, std::vector<int64_t>& axes_vec) {
+    if (node.get_input_size() == 4) {
+        auto axes = safe_cast<opset::Constant>(node.input_value(3).get_node_shared_ptr());
```

Comment:
I suppose it's a really rare case but anyway it's possible that axis input wouldn't be a constant.
I prefer to stick to previous processing scheme for the case of 4 inputs to retain non-const axis support